packages/terra-data-grid/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "terra-data-grid",
   "main": "lib/index.js",
-  "version": "0.8.1",
+  "version": "0.9.0",
```

Comment:
Should this be a 1.0 release?

cc: @JessieRandle , @cm9361

Reply:
Updated terra-data-grid to release 1.0.0